spec/unit/type/ec2_instance_spec.rb

```diff
@@ -7,13 +7,14 @@
   let :params do
     [
       :name,
-      :security_groups
     ]
   end
 
   let :properties do
     [
       :ensure,
+      :security_groups,
+      :tags,
```

Comment:
We scrapped `tags` from this commit... so feel free to put it in #53 or add a `(maint)` commit to master if you'd like.